cmd/root_options.go

```diff
@@ -36,12 +36,6 @@ func init() {
 	validate.V.RegisterStructValidation(RootOptionsLevelValidation, RootOptions{})
 }
 
-func (ro *RootOptions) SetDefaults() {
```

Comment:
Do not default the CustomBuilderImage value to builder.BaseImage.
Leave it empty when unset as we have a proper fallback.